### PR TITLE
:bug: Fix shell in Makefile

### PR DESCRIPTION
**Pull request by tmshort**
# Description

## Reviewer Checklist

- [ ] API Go Documentation
- [ ] Tests: Unit Tests (and E2E Tests, if appropriate)
- [ ] Comprehensive Commit Messages
- [ ] Links to related GitHub Issue(s)

### CODE REVIEW

**Comment by tmshort** [comment]
TL;DR: The CI was apparently using `/bin/sh`, and so, non-fatal errors occurred:
```
/bin/sh: 1: [[: not found
/bin/sh: 1: /home/runner: Permission denied
```
The `SHELL` variables was redefined to be `bash`, but it was done after some `$(shell)` lines.

**Comment by grokspawn** [comment]
Noice!  I saw those errors this morning and put them on my to-do list, but I wasn't sure where to start.

File: Makefile [hunk 1]
```diff
@@ -14,6 +14,10 @@ export IMAGE_TAG
 
 IMG := $(IMAGE_REPO):$(IMAGE_TAG)
 
+# Setting SHELL to bash allows bash commands to be executed by recipes.
```
Comment (grokspawn) [inline Makefile:17]: Do these lines need to be even higher, e.g. do `origin` calls use the shell?

Comment (everettraven) [reply Makefile:17]: Probably doesn't hurt to stick it all the way at the top anyways

Comment (tmshort) [reply Makefile:17]: `$(origin)` returns the origin of a variable; it doesn't use the shell, but I can put it right at the top.

**Review by everettraven — APPROVED** [review]

**Pull request merged**